VideoLocker/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -531,37 +522,27 @@ protected void hideSoftKeypad() {
     }
 
     public boolean showErrorMessage(String header, String message) {
-        return showErrorMessage(header, message, true);
-    }
+        hideErrorMessage();
+        alertDialog = new AlertDialog.Builder(this)
+                .setTitle(header)
+                .setMessage(message)
+                .setCancelable(false)
```

Comment:
@bguertin, according to https://openedx.atlassian.net/wiki/display/MA/Error+Handling:
"The dialog should persist until the user acknowledges it", sense the not cancelable. When I cancelable was true, the user could tap anywhere on the screen and dismiss it (maybe even accidentally).